package.json

```diff
@@ -12,12 +12,14 @@
     "dev": "turbo run dev --parallel",
     "lint": "eslint --cache --ext .js,.jsx,.ts,.tsx .",
     "prettier": "prettier --check --ignore-path .gitignore .",
-    "prettier:fix": "prettier --write --ignore-path .gitignore ."
+    "prettier:fix": "prettier --write --ignore-path .gitignore .",
+    "test": "cypress run"
```

Comment:
I think we should name this as `test:e2e` to accommodate having other types of testing in future